frontend/lib/laletterbuilder/letter-builder/my-letters.tsx

```diff
@@ -22,12 +26,145 @@ export const LaLetterBuilderMyLetters: React.FC<ProgressStepProps> = (
   );
 };
 
+interface CompletedLetterCardProps {
+  pdfBytes: string;
+}
+
+const CompletedLetterCard: React.FC<CompletedLetterCardProps> = (props) => {
+  const { children, pdfBytes } = props;
+
+  const downloadPdf = () => {
+    const bytes = atob(pdfBytes);
+    const byteChars = bytes.split("").map((el, i) => bytes.charCodeAt(i));
+    const pdfBlob = new Blob([new Uint8Array(byteChars)], {
+      type: "application/pdf;base64",
+    });
+    const pdfUrl = URL.createObjectURL(pdfBlob);
+    window.open(pdfUrl);
+  };
+
+  return (
+    <div className="jf-la-letter-card">
+      <div className="content">
+        <StaticImage
+          ratio="is-32x32"
+          src={getLaLetterBuilderImageSrc("repair-tools")}
+          alt=""
+        />
+        <h2>
+          <Trans>Notice to repair letter</Trans>
+        </h2>
+        {children}
+        {pdfBytes && (
+          <button
+            className="button is-primary is-medium jf-is-next-button"
+            onClick={downloadPdf}
+          >
+            <Trans>Download letter</Trans>
+          </button>
+        )}
+      </div>
+      <hr />
+      <Accordion question={"What's next?"} questionClassName="">
```

Comment:
Consider factoring this Accordion element out into its own component, just for ease of reading's sake.